src/SldStyleParser.ts

```diff
@@ -2181,6 +2190,75 @@ export class SldStyleParser implements StyleParser<string> {
     }];
   }
 
+  /**
+   * Translates an anchor-setting into SLD-anchor-numbers
+   */
+  getSldAnchorPointFromAnchor(anchor: TextSymbolizerAnchor, dimension: 'x' | 'y'): number {
+    if (!anchor || isGeoStylerFunction(anchor)) {
+      return 0;
+    }
+    // As explained in https://docs.geoserver.org/main/en/user/styling/sld/reference/labeling.html#anchorpoint,
+    // we have the following translation for anchors:
+    // x-dimension
+    //   left -> 0.0
+    //   center -> 0.5
+    //   right -> 1.0
+    // y-dimension
+    //   top -> 1.0
+    //   middle -> 0.5
+    //   bottom -> 0.0
+
+    if (dimension==='x') {
+      if (anchor.indexOf('left')>=0) {
+        return 0.0;
+      }
+      else if (anchor.indexOf('right')>=0) {
+        return 1.0;
+      }
+      else {
+        return 0.5;
+      }
+    }
+    else {
+      if (anchor.indexOf('bottom')>=0) {
+        return 0.0;
+      }
+      else if (anchor.indexOf('top')>=0) {
+        return 1.0;
+      }
+      else {
+        return 0.5;
+      }
+    }
+  }
+
+  /**
+   * Translates an SLD-anchor-number into a geostyler anchor-setting
+   */
+  getAnchorFromSldAnchorPoint(anchorX: any, anchorY: any): TextSymbolizerAnchor | undefined {
+    // apply defaults for missing anchor values
+    if (!anchorX){
```

Comment:
This would set an anchor with value `0` to `0.5`, since this only checks for falsey values.

I would also prefer to not set any defaults at all here, so the different rendering engines can use their respective defaults.